backend/src/controllers/contracts/invoices.js

```diff
@@ -39,4 +39,14 @@ controller.addOneWithContractId = async (request, reply) => {
   }
 };
 
+controller.findFiscalBySelectedRow = async (request, reply) => {
+  const invoiceId = Number(request.params.id);
+  try {
+    const result = await model.findFiscalBySelectedRow(invoiceId);
+    return result || controller.noQuery(reply, `The ${what.single} fiscal could not be found.`);
```

Comment:
Okay it's been a long time since I've been in these files, but does this result in something like 'The invoice fiscal could not be found?'

Reply:
it has the double pipe so it's ternary.  If result exists return it else reply with an error message